addons/hr/views/hr_employee_views.xml

```diff
@@ -89,12 +89,12 @@
                                 <field name="mobile_phone" widget="phone"/>
                                 <field name="work_phone" widget="phone"/>
                                 <field name="work_email" widget="email"/>
-                                <field name="work_location"/>
                                 <field name="company_id" groups="base.group_multi_company"/>
                             </group>
                             <group>
                                 <field name="department_id"/>
                                 <field name="parent_id"/>
+                                <field name="coach_id"/>
```

Comment:
You moved the coach_id in hr.employee view. Why not make this change in hr.employee.public and res.users view to keep consistency between these views?

Reply:
I didn't know those views existed. But I can't make the change.. This invisible attribute is awful, many files need an update and every futur person wishing to add a field there will have to think to add the invisble = 0 attribute override. This is the opposite of best practice. 
I'd rather have an empty approvers group title once in a while than the proposed solution.
I revert the changes made yesterday. I'll ask the PO if it is that important to not have the coach in the approvers

Reply:
@jbm-odoo So I just had a call with the PO. coach_id must leave this.
So, any idea how we could hide it without having to "unhide" it in the many overrides ? 
Another solution would also to get rid of it completely and recreate it in all the override. Same amount of work, but less hacky. But I don't like it either.

Reply:
I gave you my best solution. If you really think re-creating the group in all the override is prettier and the same amount of work, I'll let you do it. But in any case, it's not ok to have an empty group shown.